module-2/backend/week_one.md

```diff
@@ -5,30 +5,100 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+ Get: retreive/display info, Post: display new info, Put:save new info, Delete: delete info, Patch: update/fix?

```

Comment:
POST is used to create new
PUT is used to update
PATCH is also used to update

Generally PUT and PATCH are interchangeable, though technically believe that PUT is supposed to be used when you're looking to completely overwrite a resource, while PATCH will just update some attributes.